website/docs/litmus-installation-namespace.md

```diff
@@ -49,14 +43,13 @@ litmuschaos     https://litmuschaos.github.io/litmus-helm/
 **Note**: The chaos control plane can be placed in any namespace, though it is typically placed in "litmus".
 
 ```bash
-root@demo:~# kubectl create ns litmus
-namespace/litmus created
+kubectl create ns litmus
```

Comment:
Can we put a generic name here for namespace as it is for namespaced scope
may be like this - 
```suggestion
kubectl create ns <YOUR_NAMESPACE>
```